solidity/test/rewards/TestRewards.js

```diff
@@ -501,6 +513,31 @@ describe('Rewards', () => {
             )
         })
 
+        it("unallocates rewards allocated to terminated keeps in batch", async () => {
```

Comment:
Same here - if we want to follow the style in this file, we should put it in a separate `describe`.

Reply:
Done.